src/storage/storage.service.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import {
+    createReadStream,
+    createWriteStream,
+    promises as fsPromises,
+    write,
+} from 'fs';
+import { join } from 'path';
+import { Stream } from 'stream';
+import { promisify } from 'util';
+import { v4 as uuid } from 'uuid';
+
+const pipeline = promisify(Stream.pipeline);
+
+@Injectable()
+export class StorageService {
+    private readonly storage: string | undefined;
+    private readonly html = 'html';
+    private readonly xml = 'xml';
+
+    constructor(private readonly configService: ConfigService) {
+        this.storage = this.configService.get<string>('STORAGE');
+        if (!this.storage) {
```

Comment:
to nie ma kompletnie sensu - czemu po prostu nie zmienisz typu na `private readonly storage: string` TS ci to automatycznie zalatwi blad z przypisaniem...

Reply:
tutaj trzeba zmenić ustawienie w tsconfig, bo inaczej TS się pluje

Reply:
tak jak na discordzie to przegadalismy